src/client/cockpit-beiboot

```diff
@@ -277,7 +277,12 @@ async def run(args) -> None:
 
         # See comment in do_init() above: we tell cockpit-ws that we support
         # this and then handle it ourselves when we get the init message.
-        message.setdefault('capabilities', {})['explicit-superuser'] = True
+        capabilities = message.setdefault('capabilities', {})
```

Comment:
Shame that we can't handle this case more easily with the new json utils.... On second reading, I also sort of don't like that we modify the thing in-place.

I can't find a more elegant way, though, and then we'd need to deal with JsonError anyway...